ArduPlane/navigation.cpp

```diff
@@ -144,7 +144,10 @@ void Plane::calc_airspeed_errors()
 
     // we use the airspeed estimate function not direct sensor as TECS
     // may be using synthetic airspeed
-    ahrs.airspeed_estimate(airspeed_measured);
+    // update smoothed airspeed estimate
+    if (ahrs.airspeed_estimate(airspeed_measured)) {
+        smoothed_airspeed = smoothed_airspeed * 0.8f + airspeed_measured * 0.2f;
```

Comment:
why is this added?

Reply:
Sorry, my mind blanked on the call. 
This comes from splitting out `calc_airspeed_errors` into its own task. It was removed from `Plane::read_airspeed(void)`. My thought was by moving this out each vehicle's read_airspeed functions would be the same except for having different values for `MASK_LOG_IMU`